packages/components/atoms/f-form-field/stories/FormField.stories.js

```diff
@@ -227,3 +173,34 @@ export const FormFieldComponent = () => ({
 
 
 FormFieldComponent.storyName = 'Form Fields';
+
+FormFieldComponent.args = {
+    dropdownOptions:  [
+        {
+            disabled: true,
+            selected: true,
+            text: 'Disabled option',
+            value: ''
+        },
+        {
+            text: 'As soon as possible',
+            value: '2021-01-01T01:00:00.000Z'
+        },
+        {
+            text: 'Today in 5 minutes',
+            value: '2021-01-01T01:05:00.000Z'
+        }
+    ]
+};
+
+FormFieldComponent.argTypes = {
+    ...SharedArgTypes,
+    shouldShowLabelText:
+    {
+        control: { type: 'boolean' },
+        description: 'Select whether to show label text or not',
+        defaultValue: true
+    },
+    labelText: { description: 'no control on this story' },
```

Comment:
Looks like these are overriding the defaults, right? Does overriding them like this mean they show in the list of controls? Would setting them to `undefined` do the job?

Reply:
I want to reuse the share control file so best compromised plus doesn't affect anything.